spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/vision/CloudVisionAutoConfiguration.java

```diff
@@ -79,4 +80,10 @@ public ImageAnnotatorClient imageAnnotatorClient() throws IOException {
 
 		return ImageAnnotatorClient.create(clientSettings);
 	}
+
+	@Bean
+	@ConditionalOnMissingBean
+	public CloudVisionTemplate cloudVisionTemplate(ImageAnnotatorClient imageAnnotatorClient) {
```

Comment:
The whole auto-configuration should probably now be conditional on `CloudVisionTemplate.class` rather than `ImageAnnotatorClient.class`.